### PR TITLE
ui: update funding to daily premium to test

### PR DESCRIPTION
**Pull request by alexisgauba**
# Task:

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## FE Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

## User Facing Checklist

- [ ] I fully understand the user problem this PR is solving
- [ ] I know who the target user is for this PR and have a deep understanding of that user
- [ ] I have tried this flow thinking from the pov of the target user for this PR
- [ ] I (or working w someone on team) have scheduled a user test for this PR (if it is a large change)

### CODE REVIEW

File: packages/frontend/src/components/Charts/LongChart.tsx [hunk 1]
```diff
@@ -265,9 +264,9 @@ function LongChart() {
             Risks
           </Typography>
           <Typography variant="body2" className={classes.cardDetail}>
-            Funding is paid out of your position, similar to selling a small amount of squeeth at funding, reducing your
-            position size. Holding the position for a long period of time without upward movements in ETH can lose
-            considerable funds to funding payments.
+            Premiums are is paid out of your position, similar to selling a small amount of squeeth as you earn
```
Comment (reganomalley) [inline packages/frontend/src/components/Charts/LongChart.tsx:267]: typo here. should premiums are paid out

File: packages/frontend/src/components/Trade/Long/BreakEven.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ const BreakEven: React.FC = () => {
       <TradeInfoItem
         label="ETH move to breakeven"
         value={breakEven.toFixed(3)}
-        tooltip={`To breakeven ETH to needs to move to ${breakEven.toFixed(3)}% in one day, since current funding is 
+        tooltip={`To breakeven ETH to needs to move to ${breakEven.toFixed(3)}% in one day, since current premiums is
```
Comment (reganomalley) [inline packages/frontend/src/components/Trade/Long/BreakEven.tsx:26]: typo here. should be "since current premiums are"

**Review by reganomalley — CHANGES_REQUESTED** [review]
just a couple typos. i left comments!

**Comment by alexisgauba** [comment]
> just a couple typos. i left comments!

tyty! updated

**Review by KMKoushik — APPROVED** [review]

**Review by reganomalley — APPROVED** [review]

**Pull request merged**